src/common/adjust-body-padding.ts

```diff
@@ -1,7 +1,11 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: MIT-0
 export function adjustBodyPadding() {
-  const { height: headerHeight } = document.querySelector('#h').getBoundingClientRect();
+  const header = document.querySelector('#h');
+  if (!header) {
+    throw new Error('Header were not found');
+  }
+  const { height: headerHeight } = header.getBoundingClientRect();
   document.documentElement.style.scrollPaddingTop = `${headerHeight}px`;
 }
 
```

Comment:
This file edited manually. Auto generated one doesn't detect the header correctly. Will follow up to see why.